juriscraper/opinions/united_states/state/conn.py

```diff
@@ -72,14 +84,159 @@ def _process_html(self) -> None:
         :return: None
         """
         for row in self.html.xpath(".//*[contains(@href, '.pdf')]"):
+            date_filed_is_approximate = False
             pub = row.xpath('preceding::*[contains(., "Published")][1]/text()')
-            date = self.find_published_date(pub[0])
+            if not pub:
+                # May happen on most recent opinions, which have a header like
+                # "Publication in the Connecticut Law Journal To Be Determined"
+                date_filed = f"01/07/{self.current_year}"
+                date_filed_is_approximate = True
+
+            date_filed = self.find_published_date(pub[0])
```

Comment:
I think you should probably do something like this:

```
if not pub:
    # May happen on most recent opinions, which have a header like
    # "Publication in the Connecticut Law Journal To Be Determined"
    date_filed = f"01/07/{self.current_year}"
    date_filed_is_approximate = True
else:
    date_filed = self.find_published_date(pub[0])
```

because the date_field inside the if will always be overwritten by the date_filed below, and would probably throw an error if pub doesn't exists and tries to get the first element pub[0]

I think you can now implement the last thing you did in the make_backscrape_iterable function in AbstractSite, you may also need to update the required command in the issue to match the date format in make_backscrape_iterable

Other than this, the command was executed without problems.

Reply:
Thanks @quevon24 , you are right about that if-clause

About `make_backscrape_iterable`, this source doesn't match the pattern implemented in the base AbstractSite, so it just overrides the `make_backscrape_iterable` function and does it's own processing of the arguments

Reply:
Perfect, it's ready now, I think @flooie  is also assigned to review it. His approval is still needed to merge it.